test/net/http_test.clj

```diff
@@ -76,3 +77,26 @@
        #"SSL was required but no SSL context is present"
        (client/request nil {:request-method :get
                             :uri "https://foo.example.com"}))))
+
+(defn http-request [uri]
+  (reify io.netty.handler.codec.http.HttpRequest
+    (uri [this] uri)
+    (headers [this] (io.netty.handler.codec.http.HttpHeaders/EMPTY_HEADERS))
+    (method [this] (io.netty.handler.codec.http.HttpMethod/GET))
+    (protocolVersion [this] (io.netty.handler.codec.http.HttpVersion/HTTP_1_1))))
+
+(def invalid-uri "/news/_1200x630_crop_center-center_82_none/impfzentrum-wuerzburg-schild-fh.jpg?mtime=20210218162931&focal=30.92%%2050.13%&tmtime=20210324130011")
+
```

Comment:
found it :wink:

https://www.radiogong.com/aktuelles/news/mainfranken-das-bedeutet-der-astrazeneca-stopp-fuer-die-region

Reply:
Let's anonymize a little bit. :)